examples/hotrod/pkg/tracing/http.go

```diff
@@ -22,26 +22,30 @@ import (
 	"io"
 	"net/http"
 
-	"github.com/opentracing-contrib/go-stdlib/nethttp"
-	"github.com/opentracing/opentracing-go"
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+	"go.opentelemetry.io/otel/trace"
 )
 
 // HTTPClient wraps an http.Client with tracing instrumentation.
 type HTTPClient struct {
-	Tracer opentracing.Tracer
-	Client *http.Client
+	TracerProvider trace.TracerProvider
+	Client         *http.Client
+}
+
+func NewHTTPClient(tp trace.TracerProvider) *HTTPClient {
+	return &HTTPClient{
+		TracerProvider: tp,
+		Client:         &http.Client{Transport: otelhttp.NewTransport(http.DefaultTransport, otelhttp.WithTracerProvider(tp))},
```

Comment:
avoid long lines. Without formatting it's difficult to match which params to where, one has to count parentheses
```suggestion
		Client:         &http.Client{
			Transport: otelhttp.NewTransport(
				http.DefaultTransport, 
				otelhttp.WithTracerProvider(tp),
			),
		},
```

Reply:
Ah sorry! next time.